### DISCUSSION TITLE
installing google cloud plugin

### DISCUSSION THREAD

**Discussion by connortruex** [nocodb/nocodb#931] [Q&A]
Hey,

I am trying to install the google cloud plugin. it seems pretty straight forward. I have a bucket and a client account. I also made a key for this account, and copied the private key from the json file to nocodb. although, I am running into an issue. Everytime I try to connect, i get this error:

"error:0909006c:PEM routines:get_name: no start line"

This is what the key looks like:

"-----BEGIN PRIVATE KEY-----blahblahblah-----END PRIVATE KEY-----"

is there something I am doing wrong?

**Comment by arvindwill** [comment]
Facing the same problem

**Reply by arvindwill** [reply]
used with /n as like present in json file from GCP IAM service account and it worked now.
